src/scss/variables.scss

```diff
@@ -142,6 +144,8 @@ $themes: (
         passwordSpecialColor: #c40800,
         calloutBorderColor: $border-color-dark,
         calloutBackgroundColor: $background-color,
+        regularButtonConstrastColor: white,
```

Comment:
can't we use $background-color ? Why these `white` and `black` colors are not coming from our variables?

Reply:
> can't we use $background-color ?

This is not semantically correct as we are talking about foreground colors.

> Why these white and black colors are not coming from our variables?

I based my code on the existing `variables.scss` code and not the React styles. I fixed that, now I refer to cozy-ui css variables and edits in `variables.scss` were reverted : https://github.com/cozy/cozy-pass-web/pull/10/commits/86e410f4589b68995161f697762abdd3d718c311 and https://github.com/cozy/cozy-pass-web/pull/10/commits/8ca06e0f8c2476a5fabcc960e85e343a8ac2b2e2